### PR TITLE
Fix bug in PybelAssembler for nodes that shouldn't have variants

### PR DESCRIPTION
**Pull request by cthoyt**
Better handling for abundances and biological processes that accidentally get variants added

### CODE REVIEW

**Comment by cthoyt** [comment]
Maybe I should add some tests for this...

File: indra/assemblers/pybel/assembler.py [hunk 1]
```diff
@@ -447,8 +447,10 @@ def _get_agent_node_no_bcs(agent):
     for mut in agent.mutations:
         var = hgvs(mut.to_hgvs())
         variants.append(var)
-
-    if variants:
+    
+    if variants and not isinstance(node_data, CentralDogma):
```
Comment (bgyori) [inline indra/assemblers/pybel/assembler.py]: This is interesting - looks like a valid concern but at least in the case of some modifications, they could apply to other things too. What kind of examples did you run into where this happened?

Comment (cthoyt) [reply indra/assemblers/pybel/assembler.py]: Good question: I only found instances of GO, CHEBI, and PUBCHEM agents that had phosphorylation or acetylation modifications.

Comment (cthoyt) [reply indra/assemblers/pybel/assembler.py]: I didn't look into which GO term it was, though. That might have been interesting...

**Pull request merged**